doc/index.md

```diff
@@ -345,7 +345,7 @@ non-NULL, it is used to cache frequently used uncompressed block contents.
 #include "leveldb/cache.h"
 
 leveldb::Options options;
-options.block_cache = leveldb::NewLRUCache(100 * 1048576);  // 100MB cache
+options.block_cache = leveldb::NewLRUCache(100 * 1048576);  // 100M cache capacity
```

Comment:
The previous comment seems sufficient to me. I'll err on the side of reverting it.